### PR TITLE
Added ability to resize badge icons.

### PR DESCRIPTION
**Pull request by targeter21**
PR to address issue #40 by making the icon width and height configurable through CSS variables.

### CODE REVIEW

**Comment by targeter21** [comment]
I ran wct locally and there were no errors; looks like Travis broke for a different reason and I'm unclear on how to fix it.

**Comment by targeter21** [comment]
I was able to fix it by updating the Firefox version Travis uses to 48, the latest.

**Comment by azakus** [comment]
Firefox 48+ is using the new [geckodriver](https://github.com/mozilla/geckodriver) selenium driver, which we have been seeing issues with flaky tests in the past.

@rictic, we should evaluate if 48 is ready for primetime with geckodriver.

**Comment by rictic** [comment]
+1 to reevaluating 48. I'd be down for that on Monday.

**Comment by notwaldorf** [comment]
@targeter21 can you undo your Travis change, please? It's fine if the Firefox tests are red, it won't affect me merging the PR

File: paper-badge.html [hunk 1]
```diff
@@ -78,8 +78,8 @@
       }
 
       iron-icon {
-        --iron-icon-width: 12px;
-        --iron-icon-height: 12px;
+        --iron-icon-width: var(--badge-icon-width, 12px);
```
Comment (notwaldorf) [inline paper-badge.html]: Can you please document these properties in the `Styling` section of the docs?

**Comment by notwaldorf** [comment]
@targeter21 Does this PR fix https://github.com/PolymerElements/paper-badge/issues/40? If so, can you add that to your description, please?

File: demo/index.html [hunk 2]
```diff
@@ -107,6 +107,36 @@
...
+        </paper-badge>
+
+        <!-- Enlarged paper-badge with icon resizing. -->
+        <paper-icon-button id="icon4" icon="account-box" alt="account-box">
+        </paper-icon-button>
+        <paper-badge icon="social:mood" for="icon4" label="happy" class="large-icon">
+        </paper-badge>
+
+        <style is="custom-style">
+          paper-badge {
```
Comment (notwaldorf) [inline demo/index.html]: i think this style is actually global, so you might want to do something like `paper-badge.large` instead

File: paper-badge.html (hunk 1 above)
Comment (targeter21) [inline paper-badge.html]: Documentation added

**Comment by targeter21** [comment]
Updated the descriptions, added documentation, and changed the variables to better align with the other variable names. Also removed the Travis CI commit.

File: paper-badge.html [hunk 3]
```diff
@@ -55,6 +55,8 @@
 `--paper-badge-text-color` | The color of the badge text | `white`
 `--paper-badge-width` | The width of the badge circle | `20px`
 `--paper-badge-height` | The height of the badge circle | `20px`
+`--paper-badge-icon-width` | The width of the badge icon, allowed up to the width of the badge | `12px`
+`--paper-badge-icon-height` | The height of the badge icon, allowed up to the width of the badge | `12px`
```
Comment (targeter21) [inline paper-badge.html]: Small typo. Need to change width to height.

**Comment by notwaldorf** [comment]
Would it be possible to add a unit test, please, to make sure this doesn't break in the future?

**Comment by targeter21** [comment]
Added a wct test to verify that the iron-icon dimensions match the CSS configuration when given. The array index access seems a bit hokey, not sure if there is a better way to accomplish that.

**Comment by targeter21** [comment]
Going to go ahead and close this. If this is still a problem the code can probably be ported to the current master.

**Comment by knyto2** [comment]
Hi,

Any updates on this? I would love to get this code in so that it would resolve my issue of being able to resize the iron-icon.

**Comment by targeter21** [comment]
Hi @knyto2, right now the branch I believe had conflicts and due to inactivity, I closed it. You likely can port it to the current master in a new PR.

**Pull request closed without merging**